src/config/app-config.ts

```diff
@@ -34,11 +34,21 @@ export interface AppConfig {
 	redirects?: RedirectConfig[];
 }
 
+export const substituteEnvVariables = (key: string, value: any): any => {
```

Comment:
Litt usikker på om denne burde ligge her eller i utils. Om den ligger her, er det kanskje ikke så mye poeng å eksportere den? (mulig du må for å få testet den?)

Reply:
Kunne kanskje laget en egen json-utils.ts som har en ferdig funksjon med `JSON.parse(?, substituteEnvVariables)`.
Og at `substituteEnvVariables` også flyttes inn der.

Reply:
Må også flytte tester osv, og erstatte JSON.parse() andre steder i appen